app/controllers/application_controller.rb

```diff
@@ -1,2 +1,5 @@
 class ApplicationController < ActionController::Base
+  def current_user
+    user.first
```

Comment:
- [ ] Well done for implementing this. However, to refer to the User class  we need to rewrite `user` starting with a capital letter. This is just a Rails convention and it is how it works. Do something like this:
  
  ```ruby
  User.first
  ```

Reply:
Done!